src/chat.rs

```diff
@@ -2047,10 +2043,7 @@ async fn create_send_msg_job(context: &Context, msg_id: MsgId) -> Result<Option<
 
     let mut recipients = mimefactory.recipients();
 
-    let from = context
-        .get_config(Config::ConfiguredAddr)
-        .await?
-        .unwrap_or_default();
+    let from = context.get_configured_addr().await.unwrap_or_default();
```

Comment:
i think, here is a semantic change:

- before, errors were bubbled up and only unset addresses were converted to default
- now we continue on all errors and unset addresses

so, it is probably better to always bubble up, errors and unset addresses as we probably do not want to do a `recipients.push`with an empty string.

```suggestion
    let from = context.get_configured_addr().await?;
```